app/helpers/summary_document_helper.rb

```diff
@@ -0,0 +1,12 @@
+module SummaryDocumentHelper
+  def link_to_summary_document(appointment)
+    return if appointment.no_show? || appointment.cancelled_by_customer? || appointment.cancelled_by_pension_wise?
+    if appointment.pending?
+      link_to 'Update status before creating summary document', '', class: 'btn btn-danger', disabled: true
```

Comment:
Yeah we need to separate the instructions out away from the button label at the very least.

If we don't already, I think we should get a story written up for this so we can get the UI done properly.